app/src/main/res/layout/activity_main.xml

```diff
@@ -6,6 +6,31 @@
     android:layout_height="match_parent"
     tools:context=".MainActivity">
 
+    <androidx.appcompat.widget.Toolbar
+        android:id="@+id/toolbar"
+        style="@style/Widget.MaterialComponents.Toolbar.Primary"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        app:layout_constraintTop_toTopOf="parent"
+        app:title="@string/app_name">
+
+        <androidx.appcompat.widget.LinearLayoutCompat
+            android:layout_width="wrap_content"
+            android:layout_height="match_parent"
+            android:layout_gravity="right">
+
+            <Button
+                android:id="@+id/logInOrOutButton"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_margin="5dp"
+                android:backgroundTint="@color/black"
+                android:text="@string/title_login"
+                android:textColor="@color/white"
+                android:textSize="16dp" />
```

Comment:
I think the log in button on the app bar is kind of distracting. Have you thought about using the overflow menu from Material Design? I can help implement this since I know its not really part of your screen.

https://material.io/components/menus#dropdown-menu

![Screen Shot 2022-05-04 at 5 06 23 PM](https://user-images.githubusercontent.com/41392379/166833602-642352b5-c8ab-42de-bbe0-7195ccb84a54.png)

Reply:
Hmm, I think you're right it is kinda distracting. I'm gonna give this a go, and I'll let you know if I can't figure it out.